tests/test_mousebind.py

```diff
@@ -111,7 +111,7 @@ def test_drag_move(hlwm, x11, mouse, repeat):
     client, winid = x11.create_client()
     x, y = x11.get_absolute_top_left(client)
     # Just positioning the mouse pointer, no need to wait for hlwm
```

Comment:
The comment is not quite up to date is it? We are waiting now

Reply:
I've removed it, thanks!